iceberg/iceberg-handler/src/test/queries/positive/iceberg_branch_with_different_case.q

```diff
@@ -0,0 +1,57 @@
+-- SORT_QUERY_RESULTS
```

Comment:
i think it's better to do in UT, TestHiveIcebergBranchOperation, q tests are expensive

Reply:
Changed.